.travis.yml

```diff
@@ -11,15 +11,15 @@ go:
 before_script:
   - go version
   - export GOBIN="$GOPATH/bin"
-  - go get -u golang.org/x/lint/golint
   - go get golang.org/x/tools/cmd/goimports
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.16.0 && golangci-lint --version
```

Comment:
Is there a potential issue with the version being hard-coded?

Reply:
hard coding the version is actually the recommended method (vs tracking head)